core/pythonActionLoop/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 FROM golang:1.11 as builder
-ENV PROXY_SOURCE=https://github.com/apache/incubator-openwhisk-runtime-go/archive/golang1.11@v1.13.0-incubating.tar.gz
+ENV PROXY_SOURCE=https://github.com/apache/incubator-openwhisk-runtime-go/archive/golang1.11@1.13.0-incubating.tar.gz
 RUN curl -L "$PROXY_SOURCE" | tar xzf - \
   && mkdir -p src/github.com/apache \
   && mv incubator-openwhisk-runtime-go-golang1.11-v1.13.0-incubating \
```

Comment:
We need to remove the 'v' in this name as well, to incubator-openwhisk-runtime-go-golang1.11-1.13.0-incubating